src/main/java/uk/gov/companieshouse/pscfiling/api/controller/ValidationStatusControllerImpl.java

```diff
@@ -8,10 +8,12 @@
 import org.springframework.validation.FieldError;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestAttribute;
```

Comment:
ditto here, the validation_status request doesn't go through the Tx Interceptor either.